org.freedesktop.Sdk.Extension.mono5.appdata.xml

```diff
@@ -4,4 +4,5 @@
   <metadata_license>CC0-1.0</metadata_license>
   <name>Mono 5.x Sdk extension</name>
   <summary>Mono runtime, compiler and tools</summary>
+  <url type="bugtracker">https://github.com/flathub/org.freedesktop.Sdk.Extension.mono5</url>
```

Comment:
It needs `<url type="homepage">` as well.

Reply:
Is there a flathub.org page generated for this package?

Reply:
Homepage should point to project page, not Flathub.

Reply:
https://www.mono-project.com/ ?

Reply:
Yes.